File: module06/module025/module2/src/main/java/com/android/example/module06_module025_module2/ClassAAD.java
package com.android.example.module06_module025_module2;

public class ClassAAD {
	private 	io.reactivex.internal.schedulers.SingleScheduler instance_var_1_0 = new io.reactivex.internal.schedulers.SingleScheduler();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_1 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_2 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_3 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_4 = new io.reactivex.subscribers.TestSubscriber();

	public void method0(
		io.reactivex.internal.disposables.ListCompositeDisposable param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.subscribers.TestSubscriber<Object> param2,
		io.reactivex.subscribers.TestSubscriber<Object> param3) throws Throwable {
	}

	public void method1(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1) throws Throwable {
		for (int iAb = 0; iAb < 2; iAb++) {
			io.reactivex.schedulers.TestScheduler local_var_3_0 = new io.reactivex.schedulers.TestScheduler();
			local_var_3_0.createWorker();

			for (int iAc = 0; iAc < 1; iAc++) {
				io.reactivex.subscribers.TestSubscriber<Object> local_var_4_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_4_0.isCancelled();

				local_var_4_0.onError(new java.lang.Throwable());

				local_var_4_0.isDisposed();

			}
		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.assertSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.isDisposed();

	}

	public void method2(
		io.reactivex.subscribers.TestSubscriber<Object> param0) throws Throwable {
		param0.onError(new java.lang.Throwable());

		param0.request(914356586l);

		param0.onNext(new java.lang.Object());

	}

	public void method3(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.internal.disposables.ListCompositeDisposable param1) throws Throwable {
		io.reactivex.schedulers.TestScheduler local_var_2_2 = new io.reactivex.schedulers.TestScheduler();
		local_var_2_2.createWorker();

		param0.hasSubscription();

	}
}

File: module02/module13/module3/src/main/java/com/android/example/module02_module13_module3/ClassAAD.kt
package com.android.example.module02_module13_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAD {
	private 	val instance_var_1_0: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
	private 	val instance_var_1_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
	private 	val instance_var_1_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
	private 	val instance_var_1_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
	private 	val instance_var_1_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()

	fun method0(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		param0.isDisposed()

		for (iAb in 0..2) {
			val local_var_3_0: io.reactivex.internal.schedulers.SingleScheduler = io.reactivex.internal.schedulers.SingleScheduler()
			local_var_3_0.start()

		}
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.assertNotSubscribed()

		param0.onComplete()

	}

	fun method1(
		param0: io.reactivex.disposables.SerialDisposable
	) {
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.isCancelled()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.onError(kotlin.Throwable())

		param0.get()

	}

	fun method2(
		param0: io.reactivex.internal.disposables.ListCompositeDisposable
	) {
		param0.clear()

		val local_var_2_1: io.reactivex.schedulers.TestScheduler = io.reactivex.schedulers.TestScheduler()
		local_var_2_1.triggerActions()

	}

	fun method3(
		param0: io.reactivex.internal.disposables.ListCompositeDisposable,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param2: io.reactivex.disposables.SerialDisposable,
		param3: io.reactivex.schedulers.TestScheduler
	) {
	}

	fun method4(
		param0: io.reactivex.internal.disposables.ListCompositeDisposable
	) {
		param0.isDisposed()

		param0.isDisposed()

	}
}
